src/clj/rems/application/commands.clj

```diff
@@ -263,6 +263,13 @@
 (defn- ok [& events]
   (ok-with-data nil events))
 
+(defn- build-forms-list [catalogue-item-ids {:keys [get-catalogue-item]}]
+  (->> catalogue-item-ids
+       (mapv get-catalogue-item)
+       (mapv :formid)
```

Comment:
hmm if you did `(mapv #(select-keys % [:formid]))` here, you wouldn't need the last mapv, right?

Reply:
There's some options here but I want to rename to `:field/id`.

Reply:
oh right I missed that, nevermind